### PR TITLE
Deprecated conversion

### PR DESCRIPTION
**Pull request by aylapear**
- styled some documentation
- reviewed and updated functions Nadine noted required a double check (some were changed others were not)

### CODE REVIEW

File: R/attributes.R [hunk 1]
```diff
@@ -22,7 +24,12 @@ check_attributes <- function(x,
                              class = TRUE,
                              x_name = substitute(x),
                              error = TRUE) {
-  lifecycle::deprecate_soft("0.5.1", "check_attributes()", NULL)
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_attributes()", 
+    NULL,
+    "Pull out attributes with `attr()` and then use appropriate chk function"
```
Comment (joethorley) [inline R/attributes.R:31]: change details to "Pull out attributes with `attributes()` and then use appropriate chk function"

File: R/nchar.R [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
-  lifecycle::deprecate_soft("0.5.1", "check_nchar()", "chk::chk_match()")
+check_nchar <- function(x, 
+                        nchar = TRUE, 
+                        x_name = substitute(x),
+                        error = TRUE) {
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_nchar()", 
+    NULL, 
+    "Please use `chk::chk_equal()` with nchar() or `chk::chk_match()` instead."
```
Comment (joethorley) [inline R/nchar.R:24]: Can you change details to 
"Please use `chk::chk_equal()`, `chk::chk_range()`, `chk::chk_subset()` etc with nchar() instead."

File: R/props.R [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 #' @return An invisible copy of x (if it doesn't throw an error).
 #' @export
 check_props <- function(x, x_name = substitute(x),
                        error = TRUE) {
-  lifecycle::deprecate_soft("0.5.1", "check_props()", "chk::check_values()")
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_props()", 
+    NULL,
+    "Please use chk::check_values() with values = c(0, 1) instead")
```
Comment (joethorley) [inline R/props.R:16]: Can you change to

lifecycle::deprecate_soft(
     "0.5.1", 
     "check_props()", 
     chk::chk_range())

**Review by joethorley — CHANGES_REQUESTED** [review]
Thanks Ayla
Looking good.
Joe

File: R/nchar.R (hunk 2 above)
Comment (aylapear) [reply R/nchar.R:24]: done

File: R/props.R (hunk 3 above)
Comment (aylapear) [reply R/props.R:16]: Love this combo, we are getting at both the sum and the range of values. Great idea @nadinehussein

File: R/attributes.R (hunk 1 above)
Comment (aylapear) [reply R/attributes.R:31]: done

**Pull request merged**